transforms/helpers/util/get-telemetry-for.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs-extra");
+const path = require("path");
+const walkSync = require("walk-sync");
+const getRepoInfo = require("git-repo-info");
+const Cache = require("sync-disk-cache");
+
+const gitInfo = getRepoInfo();
+const cache = new Cache(`native-class-codemod-${gitInfo.sha}`);
```

Comment:
We should extract a util function that returns the cache instance. I'm worried about this getting out of sync (there are at least 3 spots where we call `new Cache` ATM)...